specification/billing/resource-manager/Microsoft.Billing/preview/2018-11-01-preview/billing.json

```diff
@@ -3233,6 +3280,39 @@
         }
       }
     },
+    "ValidateAddressResponse": {    
+    "type":"object",
+    "description" :" Result of the address validation",
+    "properties" : {
+      "status": {
+        "description": "status of the address validation.",
+        "$ref": "#/definitions/AddressValidationStatus"
+      },
+      "suggestedAddresses": {
+        "description": "list of suggested addresses.",
+        "type": "array",
+        "items": {
+          "$ref": "#/definitions/Address"
+        }
+      },
+      "validationMessage" : {
```

Comment:
why is validation error message part of the request body. If the validation on an address fails, shouldnt it return an error with the details part of the error body which complies with the ARM RPC error contract

Reply:
Validation error message is not part of request body. If AV fails, it returns a message in ValidationMessage property which is a part of Response object.

Reply:
sorry i meant response body itself. What i meant is -> if POST /validateAddress fails to validate an address, shouldnt the response should be a failure? In the current model, the response will be a successful response, with the validation failure in the response body.

Reply:
oh, yes, it always responds with 200, we internally call into other service which always returns 200, with validation results in response body. We apply some business logic and return 200.

Reply:
ok, thanks for confirming.